utility/src/main/java/org/oppia/util/data/UrlImageParser.kt

```diff
@@ -6,13 +6,22 @@ import android.graphics.Canvas
 import android.graphics.Rect
 import android.graphics.drawable.BitmapDrawable
 import android.graphics.drawable.Drawable
-import android.graphics.drawable.LevelListDrawable
 import android.text.Html
+import android.util.Log
 import android.widget.TextView
 import com.bumptech.glide.Glide
 import com.bumptech.glide.request.target.SimpleTarget
 import com.bumptech.glide.request.transition.Transition
 import java.net.URL
+import android.graphics.BitmapFactory
+import android.graphics.drawable.LevelListDrawable
+import android.os.AsyncTask
+import org.oppia.util.data.UrlImageParser.LoadImage
+import android.R
```

Comment:
Is this import needed? If not, please remove & other unused imports.

Reply:
removed unused import